### PR TITLE
Send current_user_time in LLM context

### PR DESCRIPTION
**Pull request by metamben**
Send `current_user_time` to the agent as specified in [Notion](https://www.notion.so/metabase/Metabot-toolbox-v1-15169354c90180cf92edd3a32f8fc62c?pvs=4#15269354c90180ddaad3c951239df235).

### CODE REVIEW

File: enterprise/backend/src/metabase_enterprise/metabot_v3/context.clj [hunk 1]
```diff
@@ -41,19 +44,25 @@
...
-  [_context]
-  (merge {}
-         #_(metabot-v3.tools.query/describe-context context)))
+  [context]
+  (-> context
+      set-user-time
+      (select-keys [:current_user_time])))
 
 (mu/defn create-reactions
   "Extracts reactions based on the current context."
```
Comment (johnswanson) [inline enterprise/backend/src/metabase_enterprise/metabot_v3/context.clj:66]: Not relevant to your change, but this is unfamiliar to me, I guess at some point we decided reactions should occur by changing the context? Are you using that functionality? It feels bizarre to have two different ways of emitting a reaction from a tool (changing the context in a particular way will auto-emit the reaction, or you can do `:reactions` in your result as well)...

**Review by johnswanson — APPROVED** [review]
LGTM, assuming we want to do this in the BE and not have it just be sent from the FE. I'm not sure there's a strong reason to do it in one or the other place.

File: enterprise/backend/src/metabase_enterprise/metabot_v3/context.clj (hunk 1 above)
Comment (metamben) [reply enterprise/backend/src/metabase_enterprise/metabot_v3/context.clj:66]: I'm not using this and I'm not aware of its origins either. I agree that two mechanisms for the same thing just lead to confusing and bugs.

**Comment by metamben** [comment]
> LGTM, assuming we want to do this in the BE and not have it just be sent from the FE. I'm not sure there's a strong reason to do it in one or the other place.

The philosophy behind this is to keep the FE as stable as possible and use the BE for translation. I like the time format the FE is sending right now (it's simple, easy to parse, and unambiguous) and the fact the agent/LLM needs something that strange as it does should not be a concern for the FE.

**Pull request merged**